public/app/features/canvas/elements/rectangle.tsx

```diff
@@ -23,23 +23,23 @@ export enum VAlign {
   Bottom = 'bottom',
 }
 
-interface TextBoxData {
+interface RectangleData {
   text?: string;
   color?: string;
   size?: number; // 0 or missing will "auto size"
   align: Align;
   valign: VAlign;
 }
 
-interface TextBoxConfig {
+interface RectangleConfig {
```

Comment:
Given newly added metric value uses these values, it may make sense to move them to the types file and just name them something generic like `TextData` / `TextConfig` as we can reuse this across multiple element types that display text